### PR TITLE
Add color palette to HIG (#1831)

### LINKED ISSUE #1831: color palette should be in the developer docs
We should probably reference and link to the <a href ="https://github.com/elementary/icons/blob/master/elementary.gpl">colour palette</a> in the docs for developers. So that devs are better equipped to make icons for their apps. It might be beneficial to explain how to use / install it as well.

### PR DESCRIPTION
**Pull request by nel-co**
Fixes #1831 

### Changes Summary

Adds a responsive color palette to the human-interface-guidelines. Preview image attached.

hig-colors

This pull request is ready for review.

### CODE REVIEW

**Comment by btkostner** [comment]
Thank you for taking time for the pull request! There are a couple of things that need to be changed before we can merge this though.

The color palette shouldn't be in the HIG, it should be on our [brand page](https://elementary.io/brand), and have a link in the HIG. When you are doing that, make sure you don't include inline styles, but instead write them in the [brand style file](https://github.com/elementary/website/blob/master/_styles/brand.css).

**Comment by danrabbit** [comment]
Hm I dunno I think the HIG might be a better place for this. I know we have colors on the brand page currently, but it seems like people designing apps and icons for elementary OS will be more interested in this than press would

**Comment by btkostner** [comment]
I was thinking more for maintainability, it would be easier to have the colors on one page, and the brand page is where that is currently.

File: docs/human-interface-guidelines.md [hunk 1]
```diff
@@ -769,7 +769,191 @@ Color, don't be afraid to use it! Many of the elementary OS icons use vibrant co
...
+		<div style="background-color:#a62100; color: white; font-size: 12px; padding: 10px; padding-left: 15px; padding-right: 15px; display: flex; justify-content: space-between;">
+		 	Orange 900 <span>#a62100</span>
+		</div>
+  </div>
+	<div style="width: 32%; flex-grow: 1; min-width: 200px; margin: 5px;">
+    <div style="background-color:#f9c440; color: #ad5f00; padding: 15px; display: flex; flex-direction: column; align-items: center;">
+			<span style="font-size: 18px;">Banana</span> <span style="font-size: 12px;">#f9c440</span>
+		</div>
+    <div style="background-color:#fff394; color: #ad5f00; font-size: 12px; padding: 10px; padding-left: 15px; padding-right: 15px; display: flex; justify-content: space-between;">
+			Banana 100 <span>#fff394</span>
```
Comment (btkostner) [inline docs/human-interface-guidelines.md]: Does not pass WCAG AA contrast test

**Comment by nel-co** [comment]
Ok, I'll change the color shortly. Am I moving them to the brand page?

**Comment by btkostner** [comment]
@nel-co It's good where it is. I would like the styles pulled out to a stylesheet though.

**Comment by nel-co** [comment]
@btkostner Alright, I'll do that. For some reason, I was thinking .md files had to be styled inline.

**Pull request merged**